code/datums/supplypacks/misc.dm

```diff
@@ -64,7 +64,8 @@
 			/obj/item/clothing/accessory/storage/black_drop_pouches,
 			/obj/item/clothing/accessory/storage/brown_drop_pouches,
 			/obj/item/clothing/accessory/storage/white_drop_pouches,
-			/obj/item/clothing/accessory/storage/webbing
+			/obj/item/clothing/accessory/storage,
+			/obj/item/clothing/accessory/storage/knifeharness
```

Comment:
This is a bit out-of-scope.

Reply:
My computer is in a cardboard box and will be for weems. Can't change it.